docs/kyma/docs/037-CRD.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Installer Custom Resource
+type: CRD
+---
+
+The installer Custom Resource (CR) file controls the Kyma installer, which is a proprietary solution based on the [Kubernetes operator](https://coreos.com/operators/). The file contains the basic information that defines Kyma installation. You can generate a custom CR file to install Kyma using specific parameters or a specific release in a `TAR.GZ` format.
+Find the custom resource template [here](../../../installation/resources/installer-cr.yaml.tpl).
+
+## Control the installation process
+
+To trigger the installation process, set the **action** label to `install` in the metadata of the Custom Resource with the installer configuration.
+To trigger the deinstallation process, set the **action** label to `uninstall` in the metadata of the Custom Resource with the installer configuration.
+
+## Generate a new Custom Resource file
```

Comment:
there is no explanation of finalizers
https://github.com/kyma-project/kyma/blob/a93fe9f83f420f2bfeb0957f34d41f087df9b3d2/installation/resources/installer-cr.yaml.tpl#L7